it/src/main/java/com/google/cloud/teleport/it/spanner/SpannerResourceManager.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.it.spanner;
+
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.SpannerException;
+
+/** Interface for managing spanner resources in integration tests. */
+public interface SpannerResourceManager {
+
+  /**
+   * Creates a table given a CREATE TABLE DDL statement.
+   *
+   * @param statement The CREATE TABLE DDL statement. Example: "CREATE TABLE Singers (" + "SingerId
+   *     INT64 NOT NULL," + " FirstName STRING(1024)," + " LastName STRING(1024)," + " SingerInfo
+   *     BYTES(MAX)" + ") PRIMARY KEY (SingerId)"
+   */
+  void createTable(String statement) throws Exception;
+
+  /**
+   * Inserts a given record into a table.
+   *
+   * @param tableRecord A mutation object representing the table record.
+   * @throws SpannerException if there is an issue inserting the table record.
```

Comment:
Once all the exceptions are cleaned up, please make sure that the `throws` comments and method signatures are in sync.